cfn-templates/sso-permission-sets.cf.yaml

```
@@ -226,3 +229,25 @@ Resources:
               - states:Get*
               - states:List*
             Resource: "*"
+  AtatDeveloperSecretReadAccess:
+    Type: AWS::SSO::PermissionSet
+    Properties:
+      InstanceArn: !Ref InstanceArn
+      Name: AtatDeveloperSecretReadAccess
+      Description: >-
+        Allow the ability to read Secrets Manager values.
+      InlinePolicy:
+        Version: 2012-10-17
+        Statement:
+          - Sid: DenyActionsInOtherAccoutns
+            Effect: Deny
+            Action: "*"
+            Resource: "*"
+            Condition:
+              StringNotEquals:
+                aws:ResourceAccount: "${aws:PrincipalAccount}"
+          - Effect: Allow
+            Action: 
+              - secretsmanager:GetSecretValue
+              - secretsmanager:ListSecrets
```

```suggestion
          - Effect: Allow
            Action: 
              - secretsmanager:GetSecretValue
              - secretsmanager:ListSecrets
              - secretsmanager:DescribeSecret
```

I would add `DescribeSecret` to this statement: https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_DescribeSecret.html